### PR TITLE
[Docs] Update installation docs

### PR DESCRIPTION
**Pull request by mcassier31**
- Creates a new page for Docker-based installation
- Updates Advanced installation topics page with Docker info
- Moves Handle Failed Jobs info in a new doc page
- Updates Getting Started readme file + file names
+ small changes in text

### CODE REVIEW

File: doc/01_Getting_Started/05_Handle_Failed_Jobs.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Handle Failed Jobs
+
+If some maintenance jobs are failed in the processing, they are discarded from the respective transport after defined retries.
```
Comment (jdreesen) [inline doc/01_Getting_Started/05_Handle_Failed_Jobs.md]:
```suggestion
If maintenance jobs fail during processing, they are discarded from the respective transport after defined number of retries.
```

Comment (mattamon) [reply doc/01_Getting_Started/05_Handle_Failed_Jobs.md]: Nice thanks! We restructured the files a bit, but we added your suggestions!

File: doc/01_Getting_Started/00_Installation/01_Webserver_Installation.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+```
+cd ./my-project
+./vendor/bin/pimcore-install
+```
+
+This launches the interactive installer with a few questions. Make sure that you set the `memory_limit` to at least `512M` in your `php.ini` file.   
+
+:::info
+
+ Pimcore allows a fully automated installation process. Read more here: [Advanced Installation Topics](../01_Advanced_Installation_Topics.md)
```
Comment (fashxp) [inline doc/01_Getting_Started/00_Installation/01_Webserver_Installation.md]:
```suggestion
 Pimcore allows a fully automated installation process. Read more here: [Advanced Installation Topics](../02_Advanced_Installation_Topics.md)
```

File: doc/01_Getting_Started/00_Installation/01_Webserver_Installation.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+Keep in mind that the cron job has to run as the same user as the web interface to avoid permission issues (eg. `www-data`).
+
+For information about how to handle failed jobs, see this [section](../01_Symfony_Messenger.md).
+
+## Caching
+Make sure to use any sort of [caching](https://pimcore.com/docs/platform/Pimcore/Development_Tools_and_Details/Cache/) to improve performance. We recommend Redis cache storage.
+
+## 6. Additional Information & Help
+
+If you would like to know more about the installation process or if you are having problems getting Pimcore up and running, visit the [Installation Guide](../23_Installation_and_Upgrade/README.md) section.
```
Comment (fashxp) [inline doc/01_Getting_Started/00_Installation/01_Webserver_Installation.md]:
```suggestion
If you would like to know more about the installation process or if you are having problems getting Pimcore up and running, visit the [Installation Guide](../../23_Installation_and_Upgrade/README.md) section.
```

File: doc/01_Getting_Started/00_Installation/01_Webserver_Installation.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+## 7. Further Reading
+
+- [Advanced Installation Topics](../01_Advanced_Installation_Topics.md)
```
Comment (fashxp) [inline doc/01_Getting_Started/00_Installation/01_Webserver_Installation.md]:
```suggestion
- [Advanced Installation Topics](../02_Advanced_Installation_Topics.md)
```

File: doc/01_Getting_Started/00_Installation/01_Webserver_Installation.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+- [Apache Configuration](../../23_Installation_and_Upgrade/03_System_Setup_and_Hosting/01_Apache_Configuration.md)
+- [Nginx Configuration](../../23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md)
+- [Database Setup](../../23_Installation_and_Upgrade/03_System_Setup_and_Hosting/05_DB_Setup.md)
+- [Additional Tools Installation](../../23_Installation_and_Upgrade/03_System_Setup_and_Hosting/06_Additional_Tools_Installation.md)
+
+Next up - [Directories Structure](../02_Directory_Structure.md)
```
Comment (fashxp) [inline doc/01_Getting_Started/00_Installation/01_Webserver_Installation.md]:
```suggestion
Next up - [Directories Structure](../03_Directory_Structure.md)
```

**Review by fashxp — CHANGES_REQUESTED** [review]
please double check the links...

**Review by fashxp — APPROVED** [review]

**Pull request merged**